lib/logstash/filters/elastic_integration.rb

```diff
@@ -454,4 +462,40 @@ def ensure_java_major_version!(minimum_major_version)
       ERR
     end
   end
+
+  ##
+  # compares the current plugin version with the Elasticsearch version connected to
+  # generates a warning or info message based on the situation where the plugin is ahead or behind of the connected Elasticsearch
+  def check_versions_alignment
+    plugin_version_parts = VERSION.split('.')
+    plugin_major_version = plugin_version_parts[0].to_i
+    plugin_minor_version = plugin_version_parts[1].to_i
+
+    base_message = "This #{VERSION} version of plugin embedded Ingest node components from Elasticsearch #{plugin_major_version}.#{plugin_minor_version}"
+    logger.info(base_message)
+
+    es_version_parts = connected_es_version_info["number"].split('.')
+    es_major_version = es_version_parts[0].to_i
+    es_minor_version = es_version_parts[1].to_i
```

Comment:
```suggestion
    es_major_version, es_minor_version = connected_es_version_info["number"].split('.').map(&:to_i)
```

Reply:
Included in [`b61b374`](https://github.com/elastic/logstash-filter-elastic_integration/pull/247/commits/b61b374204d3e5e0a7316501e3f4f8e5c997c35c)